2.4/test/run

```diff
@@ -145,11 +145,34 @@ function test_config_option() {
     docker stop $(get_cid ${name})
 }
 
+function test_text_search() {
```

Comment:
I like @omron93 suggestion... I don't think we need to build complex test, the point of my test was just to verify nothing breaks when you set that variable up and mongo will start successfully.